WebDevStudios/wds-acf-blocks
# Feature / Adds Container Classes

### DESCRIPTION ###
- Adds container classes to blocks so that background colors expand full width but content remains contained
- Requires this related PR from wd_s https://github.com/WebDevStudios/wd_s/pull/936

### SCREENSHOTS ###

https://user-images.githubusercontent.com/3207552/193116761-d5a5d46d-8cbc-4d54-85e5-cd40d4b1227b.mov



### OTHER ###

### STEPS TO VERIFY ###
- Checkout `feature/conditional-container-class` in `wd_s`
- Add blocks to page and add background
- Confirm background is full width and content is contained
